generator/util/generateCode.py

```diff
@@ -70,35 +63,40 @@ def generate_code_for(filename, overwrite=True):
     """
 
     # Attempt to parse the XML file:
-    global_variables.running_tests = False
+    gv.running_tests = False
     parser = ParseXML.ParseXML(filename)
-    ob = []
-    if global_variables.code_returned == \
-            global_variables.return_codes['success']:
+    ob = []  # The big structure of dictionaries etc created from the DOM.
+    if gv.code_returned == gv.return_codes['success']:
         # catch a problem in the parsing
         try:
             ob = parser.parse_deviser_xml()
-        except:
-            global_variables.code_returned \
-                = global_variables.return_codes['parsing error']
-    if global_variables.code_returned == \
-            global_variables.return_codes['success']:
+        except Exception:
+            gv.code_returned = gv.return_codes['parsing error']
+    if gv.code_returned == gv.return_codes['success']:
         name = ob['name'.lower()]
-        language = global_variables.language
+        language = gv.language
         # TO DO REMEMBER TO REMOVE   TODO why?
```

Comment:
I think my note to remove it relates to the fact that it should be a try/catch but I removed it when I was testing and should add it back if doing a release.

Reply:
So @skeating , should I add it back in?

Reply:
As agreed, I have uncommented the try/catch block and all tests continued to work, so I'm going to mark this as resolved and then merge this PR.